.github/workflows/examples-k32w.yaml

```diff
@@ -64,10 +70,14 @@ jobs:
                       ${{ env.BUILD_TYPE }}-example-build-${{
                       steps.outsuffix.outputs.value }}
                   path: |
-                      out/lock_app_debug/chip-k32w061-lock-example.out
-                      out/lock_app_debug/chip-k32w061-lock-example.out.map
-                      out/lighting_app_debug/chip-k32w061-light-example
-                      out/lighting_app_debug/chip-k32w061-light-example.map
+                      out/lock_app_release/chip-k32w061-lock-example.out
+                      out/lock_app_release/chip-k32w061-lock-example.out.map
+                      out/lighting_app_release/chip-k32w061-light-example
+                      out/lighting_app_release/chip-k32w061-light-example.map
+                      out/lock_app_se_release/chip-k32w061-lock-example.out
```

Comment:
Could we pick as single output for artifacts? we use these only for bloat diff against versions, however I expect bloat will be similar between things like lock and lighting.

Our quota is quite small as these outputs are often 1MB+.

Reply:
Sure. Now it is only one binary for analysis. The Lighting binary